lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,235 @@
+package Test::Pod::CoverageChange;
+# ABSTRACT: Test Perl files for POD coverage and syntax changes
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+# AUTHORITY
+
+use utf8;
+
+=encoding utf8
+
+=head1 NAME
+
+Test::Pod::CoverageChange - Test Perl files for POD coverage and syntax changes
+
+=head1 SYNOPSIS
+
+    use Test::Pod::CoverageChange qw(pod_coverage_syntax_ok);
+
+    Test::Pod::CoverageChange::pod_coverage_syntax_ok('lib', {
+        Module::With::3::expected::naked::subs              => 3,
+        AnotherModule::With::10::expected::naked::subs      => 10,
+        YetAnotherModule::With::1::expected::naked::subs    => 1,
+        YetAnotherModule::With::5::expected::naked::subs    => 5,
+    }, [
+        We::Ignore::ThisModule,
+        We::Also::Ignore::This::Module
+    ]);
+
+=head1 DESCRIPTION
+
+C<Test::Pod::CoverageChange> is a helper combining L<Test::Pod::Coverage> and
+L<Pod::Checker> to test for both POD coverage and syntax changes for a module
+distribution at once, via a single call to L</pod_coverage_syntax_ok>.
+
+C<passes> if the file have no POD syntax or coverage error.
+C<fails> if latest changes increased/decreased numbers of naked sub for the packages that have allowed naked subs.
+C<fails> if a package allowed to have naked subs has 100% POD coverage.
+C<fails> if a file in a given path has POD syntax error or has no POD.
+
+Ignores to check every package that we pass as C<$ignored_package>
+
+=cut
+
+use Test::More;
+use Pod::Checker;
+use Pod::Coverage;
+use File::Find::Rule;
+use Test::Pod::Coverage;
+use Module::Path qw(module_path);
+use List::Util qw(any);
+
+use constant {
+    POD_SYNTAX_IS_OK => 0,
+    FILE_HAS_NO_POD  => -1,
+    FALSE            => 0,
```

Comment:
`FALSE` and `TRUE` constants like this really don't help at all - 0 and 1 are always false/true in Perl, just those those directly.

Reply:
Done.